disqus comments on blog posts

lboeman opened the pull request.

wholmgren commented:
Thanks. Let's let this simmer while we come up with a comprehensive communications strategy.

wholmgren commented:
I should add that as of now, I'm inclined to keep the blog part of this and dump the disqus part.

For reference: https://lboeman.github.io/SolarArbiter.github.io/2018/10/25/the-day-before-disqus-was-added.html (note that css is messed up on the fork and that's unrelated)

wholmgren commented:
@lboeman trying to summarize our conversation:

1. remove disqus
2. add date to title line. Is there also an author field?
3. figure out if there is a standard way to put images in jekyll blogs
4. building from this branch, I will write a post with some real content. Probably plans for what's coming up in the stakeholder engagement process and some explanation of the existing content.

Did I miss anything?

lboeman commented:
@wholmgren I'm also adding some pagination and an 'excerpt' on the blog listing page but otherwise, sounds good.

lboeman commented:
@wholmgren To answer the images question: Images are just linked to in the root `/images` folder (or whatever we want to call it) [Jekyll docs](https://jekyllrb.com/docs/posts/#including-images-and-resources)

lboeman commented:
I updated the blog listing page and created a layout for posts to automatically print the post title, author and date. 
@wholmgren you should be able to update the blog post in this PR and just change the author.
I've also just changed my fork to be hosted at lboeman.github.io with no /SolarArbiter.github.io so you can actually see the requests. The blog listing page is [here](https://lboeman.github.io/blog/)

wholmgren commented on _layouts/blog.html:
```diff
@@ -0,0 +1,26 @@
+    <main role="main" class="flex-shrink-0">
+      <div class="container mt-4">
+	    <h2>{{ page.title }}</h2>
+	    <p><span class="font-weight-bold author">{{ page.author }}, </span><i> {{ page.date | date: "%b %d, %y" }}</i></p>
```
change %y to %Y to render as 2018 instead of 18?

lboeman replied:
:+1:

wholmgren commented on _includes/disqus.html:
```diff
@@ -0,0 +1,12 @@
+})();
+</script>
+<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
+{% endif %}
```
I thought we were dropping disqus for now.

lboeman replied:
Yes, I forgot to remove the includes, but removed the includes statements. Fixed this.

wholmgren commented:
@alorenzo175 happy (enough) with this?
